### PR TITLE
Added a caption for anshubhatis.md

### PR DESCRIPTION
**Pull request by anshubhatia**
Change Request - I added caption to the read file

## Caption added by Anshu Bhatia
- [ ] Text

### CODE REVIEW

File: _posts/0000-01-17-anshubhatia.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Anshu Bhatia - GIT
```
Comment (octowilliam) [inline _posts/0000-01-17-anshubhatia.md:6]: Emoji?

File: _posts/0000-01-17-anshubhatia.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (octowilliam) [inline _posts/0000-01-17-anshubhatia.md:6]: I'm writing a comment about the caption 🎇

**Review by octowilliam — CHANGES_REQUESTED** [review]
@anshubhatia I made some comments

**Review by octowilliam — APPROVED** [review]

**Comment by anshubhatia** [comment]
Merged --Anshu Bhatia

**Pull request merged**